src/sentry/web/urls.py

```diff
@@ -438,13 +437,6 @@ def init_all_applications():
         react_page_view,
         name='sentry-organization-stats'
     ),
-
-    # TODO REMOVEME #NEW-SETTINGS, redirect to team settings?
-    url(
-        r'^organizations/(?P<organization_slug>[\w_-]+)/teams/(?P<team_slug>[\w_-]+)/remove/$',
-        RemoveTeamView.as_view(),
-        name='sentry-remove-team'
```

Comment:
I don't think this will be an issue, but to be on the safe side can we redirect this route to `/settings/:org/teams/:team/settings/`?

Reply:
Done :+1: